app/views/redis_log/_header.html.erb

```diff
@@ -6,7 +6,7 @@
 <div class="pull-right">
   <code>
     <span style="display:inline-block;">
-      <%= Time.at(req[:timestamp]).iso8601 %>
+      <%= Time.at(req[:timestamp]).strftime("%-y/%-m/%-d %-H:%M:%S.%-3N") %>
```

Comment:
Why this format, rather than keeping it ISO 8601?

Reply:
It's more compact -- timezone isn't terribly important, and I'm trying to conserve space in the header bar.

Reply:
Can we do pseudo-ISO, in that case? This format is kinda hard to read. What about `%Y-%m-%d %H:%M:%S`?

Reply:
I kinda like nanoseconds, but I don't care too much. Just make sure it still looks pretty on all widths -- I remember fighting with some CSS related to overflow.

Reply:
We can add milliseconds too if you want 'em. My concern is more about the non-standardness of two-digit years and one-digit months/dates/hours, etc. It violates the whole "least surprise" thing.

Reply:
The important thing is that 1) any timestamp will have the same number of characters (so it looks aligned) and 2) the timestamps will be minimal to leave room for other info. Beyond that, I don't have strong feelings.

Reply:
`%Y-%m-%d %H:%M:%S` (with optional `.%-3N`) works for those